framework/docker.go

```diff
@@ -112,6 +112,51 @@ func NewDockerClient() (*DockerClient, error) {
 	return &DockerClient{cli: cli}, nil
 }
 
+// ExecContainer executes a command inside a running container by name and returns the combined stdout/stderr.
+func (dc *DockerClient) ExecContainer(containerName string, command []string) (string, error) {
+	L.Info().Strs("Command", command).Str("ContainerName", containerName).Msg("Executing command")
+	ctx := context.Background()
```

Comment:
how about using ctx with some timeout?

Reply:
It doesn't make sense to me right now since our tests do not use context everywhere, so they'll provide context.Background() but put it as a ticket so we can add it later for all our utilities https://smartcontract-it.atlassian.net/browse/DX-481